server.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const path = require('path');
+
+app = express();
+
+app.use(express.static(path.join(__dirname, 'dist')));
+
+app.use('*', function (req, res) {
+  res.sendFile(path.join(__dirname, 'dist/index.html'));
+});
+
+
+const port = 3000;
```

Comment:
Здесь можно порт указать таким образом:
```suggestion
const PORT = process.env.PORT || 3000; 
```
Тогда будет возможность брать значение из env-переменных, а уже если там её не окажется, то брать значение по умолчанию